src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -247,9 +247,13 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
 
     with self.argument_context('storage account blob-service-properties update',
                                resource_type=ResourceType.MGMT_STORAGE) as c:
+        from ._validators import validator_delete_retention_period
         c.argument('account_name', acct_name_type, id_part=None)
         c.argument('resource_group_name', required=False, validator=process_resource_group)
         c.argument('enable_change_feed', arg_type=get_three_state_flag(), min_api='2019-04-01')
+        c.argument('delete_retention', arg_type=get_three_state_flag(), arg_group='Soft Delete', min_api='2018-07-01')
```

Comment:
please change arg_group name to “Delete Retention Policy"

Reply:
Done